.github/workflows/main.yaml

```diff
@@ -42,17 +42,19 @@ jobs:
         run: npm install
         
       - name: tsc
-        run: npm run build
+        run: npm run build -w packages
```

Comment:
Wouldn't `npm run build` run the same, as it basically execute `npm run build -w packages`?

Reply:
Yes it does, will change it.